src/admin/redux/modules/letters.js

```diff
@@ -0,0 +1,57 @@
+const { createAction } = require('redux-actions')
+const { get_body } = require('app/http')
+const { merge, compose, objOf, map, props, zipWith, pick } = require('ramda')
+const formatDate = require('app/format_date')
+
+const SEND_NEWSLETTER_POST =
+  'SEND_NEWSLETTER_POST'
+const SEND_SUB_REMINDER_POST =
+  'SEND_SUB_REMINDER_POST'
+
+const initialState = {
+  post_members: [],
+  sub_reminders: []
+}
+
+const reducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+  case SEND_NEWSLETTER_POST:
+    return { ...state, post_members: payload.results }
+  case SEND_SUB_REMINDER_POST:
+    const idObj = map(pick([ 'id' ]), payload.results)
+    const addressArray = map(addressArr, payload.results)
+    const contentArray = map(objOf('email_content'), map(inject, payload.results))
+    const letterObj = zipWith(merge, contentArray, addressArray)
```

Comment:
We just tried to log the following line:
reduce(zipWith(merge), {}, [idObj, contentArray, addressArray])

But we got an empty array/obj

Does the initial value have to be an array of objects? The following did give us back an array of populated objects, but it seems like we would have to populate the initial array with the correct number of objects (ie the number of users in the response.payload):
reduce(zipWith(merge), [{}], [idObj, contentArray, addresses])